model/task/db.go

```diff
@@ -160,6 +160,32 @@ var (
 		},
 	}
 
+	sortDisplayName = bson.M{
+		"$sort": bson.M{
+			"display_name": 1,
+		},
+	}
+
+	project = bson.M{
+		"$project": bson.M{
+			"_id":           0,
+			"build_variant": "$" + BuildVariantKey,
+			"display_name":  "$" + BuildVariantDisplayNameKey,
```

Comment:
I hesitate to call this display_name since it's for the build variant; the naming should reflect that